src/node_types.cc

```diff
@@ -0,0 +1,69 @@
+#include "node_internals.h"
+
+using v8::Context;
+using v8::FunctionCallbackInfo;
+using v8::Local;
+using v8::Object;
+using v8::Value;
+
+namespace node {
+namespace {
+
+#define VALUE_METHOD_MAP(V)                                                   \
+  V(External)                                                                 \
+  V(Date)                                                                     \
+  V(ArgumentsObject)                                                          \
+  V(BooleanObject)                                                            \
+  V(NumberObject)                                                             \
+  V(StringObject)                                                             \
+  V(SymbolObject)                                                             \
+  V(NativeError)                                                              \
+  V(RegExp)                                                                   \
+  V(AsyncFunction)                                                            \
+  V(GeneratorFunction)                                                        \
+  V(Promise)                                                                  \
+  V(Map)                                                                      \
+  V(Set)                                                                      \
+  V(MapIterator)                                                              \
+  V(SetIterator)                                                              \
+  V(WeakMap)                                                                  \
+  V(WeakSet)                                                                  \
+  V(ArrayBuffer)                                                              \
+  V(TypedArray)                                                               \
```

Comment:
We no longer need this with `isTypedArray` being present in `lib/internal/types.js`.